src/Portal/Portal.tsx

```diff
@@ -65,16 +65,33 @@ export const Portal: React.FC<React.PropsWithChildren<PortalProps>> = ({
   onMount,
   containerName: _containerName,
 }) => {
-  const hostElement = document.createElement('div')
+  const [element, setElement] = useState<HTMLDivElement | null>(() => {
+    // If we're in SSR, return null so that nothing is rendered
+    if (typeof document === 'undefined') return null
+    return document.createElement('div')
```

Comment:
when we aren't in SSR, we'll create this element immediately, and use it - avoiding the need to render null at all, but when we do SSR, we'll render null.

this could create a server mismatch, in which case we might prefer always defaulting to null - not quite sure how we'd validate that currently

Reply:
Was hoping this worked, but in tsting it's not quite there.